test/feature/finding_controllers.feature

```diff
@@ -48,30 +48,7 @@ Feature: Finding controllers
           res.end 'user list'
       module.exports = UsersController
       """
-    And an exprestive app with the option "controllersDirPath" set to `"./my_controllers"`
-    When making a GET request to "/users"
-    Then the response code should be 200
-    And the response body should be "user list"
-
-
-  Scenario: an application with a restrictive controller whitelist
-    Given a file "controllers/not_a_controller.coffee" with the content
-      """
-      throw 'not a controller'
-      """
-    And an exprestive app with the option "controllersWhitelist" set to `/^(?!not_a_controller).coffee$/`
-    Then the app doesn't error
-
-
-  Scenario: an application with a different controller file name format
```

Comment:
I think we only need one test for the custom pattern so removed this test as well